modules/gitops-release-notes-1-12-2.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-2_{context}"]
+= Release notes for {gitops-title} 1.12.2
+
+{gitops-title} 1.12.2 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.2_{context}"]
+== Errata updates
+
+[id="RHSA-2024:2816-gitops-1-12-2-security-update-advisory_{context}"]
+=== RHSA-2024:2816 - {gitops-title} 1.12.2 security update advisory
+
+Issued: 2024-05-09
```

Comment:
```suggestion
Issued: 2024-05-10
```

Reply:
@Dhruv-Soni11 change the Errata issue date to May 10

Reply:
Yes, please.

Reply:
Fixed.